packages/gafl-webapp-service/src/pages/licence-details/licence-length/update-transaction.js

```diff
@@ -86,6 +87,8 @@ export default async request => {
   const { payload } = await request.cache().helpers.page.getCurrentPermission(LICENCE_LENGTH.page)
   const permission = await request.cache().helpers.transaction.getCurrentPermission()
   permission.licenceLength = payload['licence-length']
+
+  await findPermit(permission, request)
```

Comment:
Not your fault but the passing in of variables to be modified is a bit grotty; I think we should have a tech debt ticket to change this

Reply:
Logged as IWTF-3375 👍